docs/_drafts/2017-05-01-ReleaseMvvmCross5.md

```diff
@@ -11,24 +11,36 @@ For the last 6 months we have been working on this release and we're really exci
 
 Let's have a look at a couple of things:
 
-## Highlights
+# Highlights
 
-### Merge of repos
+## Merge of repos
 For a long time we have been looking at merging the different repositories of the project.
 Besides a better overview one of the main advantages this brings is the ability to setup a proper CI process.
 This is hugely beneficial in the release process itself and makes (patch) releasing MvvmCross a breeze. 
 
-### New website with improved documentation
+## New website with improved documentation
 Over time we received a lot of feedback from developers that use MvvmCross on a day-2-day basis. Besides the usual
 hugs and kisses the most common remark we often got was: "sir, you need to improve the documention". And as you all know: writing documention is hard. And not always the most funny thing you can think of. So our first focus was enabling *you* to help out improving the documentation. With 5.0 the documentation has landed in the GIT repo, making it possible to submit documentation (changes) just as you do with code: create a pull request. We're already seeing the benefits of this: the amount of community driven documentation changes has increased tenfold. 
 
 Over the coming months we'll introduce a 'documentation policy' to make sure the documentation keeps on improving over time and keeps it uniformity.
 
-### New iOS presenter
+## Open Collective
+
+As you all know MvvmCross is an Open Source project, so that means we're not making any money out of it. But sometimes we're facing actual costs which is always difficult to arrange. To improve on this situation we've created the MvvmCross Open Collective - a place where you can donate your money to the project but also have full insight to what we're actually doing with it. We really hope you're going to join this Open Collective!
+
+https://opencollective.com/mvvmcross
+
+## Improved support for Xamarin.Forms
+
+In MvvmCross 5.0 we added much improved support for Xamarin.Forms! We added MvvmCross specific Pages, App classes, and Setup for Forms to enable Bindings, Ioc, DI and much more! Make sure to use the new base classes to enable Forms support.
+
+Find all the details in the updated [documentation](https://www.mvvmcross.com/documentation/platform/xamarin-forms)!
```

Comment:
We should definitely improve it later because its the most searched term on the website. For now i think this shouldn't stop the release!